client/lib/react-test-env-setup/test/index.js

```diff
@@ -9,7 +9,7 @@ var jsdom = require( 'jsdom' ).jsdom,
  */
 var setupEnvironment = require( '../' );
 
-describe( 'react-test-env-setupEnvironment', function() {
+describe( 'react-test-env-setup', function() {
 	afterEach( function() {
 		delete global.window;
```

Comment:
>Probably we should sync `afterEach` with what we do in [cleanup function](https://github.com/Automattic/wp-calypso/blob/update/single-test-runner-ms-rtes-pm/client/lib/react-test-env-setup/index.js#L39). There are more global variables removed.

Good call. The cleanup function didn't exist when the tests were originally written, but definitely makes sense to use now. Used in the rebased 62985b25b2c7dcc89aa3a84c1d303229a0fdeefa.